src/TimePicker/TimePickerDialog.js

```diff
@@ -86,6 +86,10 @@ class TimePickerDialog extends React.Component {
       root: {
         fontSize: 14,
         color: this.getTheme().clockColor,
+        WebkitUserSelect: 'none',
+        KhtmlUserSelect: 'none',
+        MozUserSelect: 'none',
```

Comment:
@oliviertassinari : I just added them to be on a safer side w.r.t browser compatibility. I can remove it, if needed. Let me check it anyways.